libs/components/src/lib/avatar/index.tsx

```diff
@@ -1,13 +1,103 @@
-import React from "react";
+import React, { useCallback, useState } from "react";
 import Image from "next/image";
 import { PencilIcon } from "@heroicons/react/solid";
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import { storage } from "../../../../../apps/frontend/services/firebase";
```

Comment:
can you move the firebase setup into the components library so you dont have to do this